src/game/client/shadow/w3dvolumetricshadow.cpp

```diff
@@ -0,0 +1,512 @@
+/**
+ * @file
+ *
+ * @author tomsons26
+ *
+ * @brief
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3dvolumetricshadow.h"
+#include "hlod.h"
+#include "mesh.h"
+#include "meshmdl.h"
+#include "missing.h"
+#include <cmath>
+#include <new>
+
+class MeshModelClass;
+
+W3DShadowGeometryMesh::W3DShadowGeometryMesh() :
+    m_polyNeighbors(nullptr),
+    m_numPolyNeighbors(0),
+    m_parentVerts(nullptr),
+    m_polygonNormals(nullptr),
+    // BUGFIX Init all members
+    m_mesh(nullptr),
+    m_modelIndex(0),
+    m_verts(nullptr),
+    m_parentGeometry(nullptr),
+    m_numVerts(0),
+    m_numPolygons(0),
+    m_polygons(nullptr)
+{
+}
+
+W3DShadowGeometryMesh::~W3DShadowGeometryMesh()
+{
+    Delete_Neighbors();
+
+    if (m_parentVerts != nullptr) {
```

Comment:
remove. These do not exist. When you delete[] or delete it implicitly does a nullptr check all you are doing here is adding a unneeded branch.